specs/core/0_beacon-chain.md

```diff
@@ -680,7 +678,7 @@ def deposit(deposit_input: bytes[2048]):
         self.deposit_tree[index] = sha3(concat(self.deposit_tree[index * 2], self.deposit_tree[index * 2 + 1]))
 
     self.deposit_count += 1
-    if msg.value == as_wei_value(MAX_DEPOSIT, "ether"):
```

Comment:
this may be more than a one-liner now but we want to render `MAX_DEPOSIT` as a wei value here.

Reply:
I changed it to `as_wei_value(MAX_DEPOSIT, "gwei")`